components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -565,6 +565,11 @@ lnd/clm2/surfdata_map/surfdata_conusx4v1_simyr2000_c160503.nc</fsurdat>
 <flanduse_timeseries hgrid="ne30np4"       rcp="2.6" sim_year_range="1850-2100"
  use_crop=".false." >lnd/clm2/surfdata_map/surfdata.pftdyn_ne30np4_rcp2.6_simyr1850-2100_c130524.nc</flanduse_timeseries>
 
+<!-- Land Use Harmonization unified data sets for dynamic FATES land use change -->
+<fluh_timeseries hgrid="4x5" sim_year_range="1850-2015" use_fates=".true."
+ >lnd/clm2/surfdata_map/fates-sci.1.68.3_api.31.0.0_tools.1.0.1/LUH2_states_transitions_management.timeseries_4x5_hist_simyr1850-2015_c231101.nc</fluh_timeseries>
```

Comment:
1. This file is not on the E3SM input data server.
2. FATES's SCI/API version number will continue to get updated. This would mean we would have many `fates-sci*` directories. How we have a single dir (e.g. `lnd/clm2/surfdata_map/fates`) and the metadata of the LUH2 file can include the information regarding the SCI/API version. In the F90 code, you can also include a sanity check regarding the SCI/API version.

Reply:
1. @bishtgautam this file is fairly large.  I submitted a request about 2 weeks ago to get access to the ANL servers using the procedure laid out on confluence so that I could load this and future updates directly.  I haven't seen movement on that request.  In the meantime, the data is on `perlmutter`; is this something you could upload for me via globus?
2.  That should work.  The file structure here was the recommended by ctsm folks, although I can't remember the rationale currently.  I'll definitely add the fates sci/api version to the data file meta data.  That said, the meta data check isn't really needed for usage here.